src/main/java/com/azure/cosmos/examples/bulk/async/SampleBulkQuickStartAsync.java

```diff
@@ -7,10 +7,7 @@
 import com.azure.cosmos.examples.common.AccountSettings;
 import com.azure.cosmos.examples.common.Families;
 import com.azure.cosmos.examples.common.Family;
-import com.azure.cosmos.implementation.ImplementationBridgeHelpers;
-//  <CosmosBulkOperationsImport>
```

Comment:
Is this not used in any of our docs?

Reply:
hmm, I am not sure how this is being used - but I reverted it back to avoid breaking other places. Good catch thanks :)

Reply:
Seems to be used in https://github.com/MicrosoftDocs/azure-docs-pr/blob/b8139b28027ac565564a3e6605975a7e5e344a98/articles/cosmos-db/nosql/how-to-migrate-from-bulk-executor-library-java.md

Reply:
So good that you reverted it or that article was busted :) 

https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/how-to-migrate-from-bulk-executor-library-java

![image](https://user-images.githubusercontent.com/1633401/197630836-56f68939-eb5c-44c3-99aa-49b9f350ae4e.png)

Reply:
thanks @ealsur for finding it :)